elastic/kibana
# [Security Solution] Flyout Insights - threats

## Summary

This PR adds the following panel to expandable flyout:

![image](https://user-images.githubusercontent.com/11671118/234833889-2e09f4bb-eda6-4f63-ba79-654648dda2ed.png)

### How to test this
add `xpack.securitySolution.enableExperimental: ['securityFlyoutEnabled']` to the kibana.json file
run `yarn es snapshot --license trial, yarn test:generate and yarn start --no-base-path`

Navigate to alerts  (see the data setup here https://github.com/elastic/security-team/issues/6422)
and expand the flyout for an event created with threat match rule.

### Checklist

Delete any items that are not applicable to this PR.
